src/content/concepts/index.md

```diff
@@ -12,6 +12,8 @@ contributors:
 
 At its core, *webpack* is a _static module bundler_ for modern JavaScript applications. When webpack processes your application, it recursively builds a _dependency graph_ that includes every module your application needs, then packages all of those modules into one or more _bundles_.
 
+T> Learn more about JavaScript modules and webpack modules [here](/modules).
```

Comment:
Shouldn't the link be to `/concepts/modules` -- e.g. [here](https://webpack.js.org/concepts/modules).